.gitignore

```diff
@@ -24,6 +24,7 @@ test-cmds
 # Generated assets
 pkg/packagekit/internal/assets.go
 pkg/packagekit/wix/internal/assets.go
+pkg/packaging/assets/tuf/root.json
```

Comment:
Why are we ignoring two root.jsons here?

Reply:
The packaging code and the tuf code both need a validated root.json. (The packaging code uses it to get latest validated TUF metadata, so that it can pick the correct version for the given channel.)

Reply:
(Also answers https://github.com/kolide/launcher/pull/2142#discussion_r1998947297)